BUILD

```diff
@@ -8,6 +9,7 @@ cc_library(
     name = "include",
     hdrs = glob(["include/proxy-wasm/**/*.h"]),
     deps = [
+        "@com_google_absl//absl/container:flat_hash_map",
```

Comment:
let's not use absl and stick to std as I commented here https://github.com/proxy-wasm/proxy-wasm-cpp-host/pull/89#discussion_r543744201